silverback/cluster/types.py

```diff
@@ -374,3 +378,29 @@ class BotInfo(BaseModel):
     registry_credentials_id: str | None
 
     environment: list[EnvironmentVariable] = []
+
+
+class LogLevel(IntEnum):
```

Comment:
Silverback uses ape's logging?

Reply:
Wasn't aware ape had its own enum for this. Done.